services/ort-run/src/main/kotlin/OrtRunService.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
```

Comment:
I am fine with moving this functionality from `workers:common` to `services`, but the existing two `OrtRunService`s seem rather disjunct to me. One provides functions dealing with multiple runs, the other one deals with the data of a single run (and maybe should be better named `OrtResultService`?). So, is it really beneficial to merge the two? The resulting class becomes quite huge with a long list of constructor parameters, and I don't see much synergy of functionality that can be shared.

Reply:
I planned to split the big class by run type, so move all functions related to an analyzer run to an `AnalyzerRunService` and so on, and keep only functions that deal with the whole run in the `OrtRunService`, like `generateOrtResult`, or functions that do not belong to a specific run, like `storeResolvedPackageCurations` or `storeIssues`. What do you think?

Reply:
Ok, it would probably be possible to have a common `WorkerRunService` base class defining a common interface and providing functionality based on the `WorkerJobRepository` interface. The workers would then use their dedicated `XXXRunService`. Is this the plan?